python/adelphi/adelphi/export.py

```diff
@@ -98,4 +98,7 @@ def get_common_metadata(self, cluster, props):
         metadata.update(self.get_cluster_metadata(cluster))
         metadata["creation_timestamp"] = datetime.now(utc).isoformat()
         return metadata
-        
+
+    # Base impl... can be overridden if necessary
+    def each_keyspace(self, ks_fn):
+        ks_fn(self.keyspace, self.keyspace_id)
```

Comment:
This change (plus the removal of this explicit default impl from the subclasses who are now just using the superclass default) isn't really related to nosqlbench support... it's just a thing I noticed while working on the code.